velox/functions/lib/window/NthValueBase.h

```diff
@@ -13,56 +13,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#pragma once
 
 #include "velox/common/base/Exceptions.h"
 #include "velox/exec/WindowFunction.h"
-#include "velox/expression/FunctionSignature.h"
 #include "velox/vector/FlatVector.h"
 
 namespace facebook::velox::functions::window {
 
-namespace {
-class NthValueFunction : public exec::WindowFunction {
+class NthValueFunctionBase : public exec::WindowFunction {
```

Comment:
Its Velox convention to have the class name same as the fileName. Please can you rename this class.

Reply:
Updated, thanks.